data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Бэкон", "Bacon" ],
+      [ "Бальдерас", "Balderas" ],
+      [ "Банкс", "Banks" ],
+      [ "Барретт", "Barrett" ],
+      [ "Барриентос", "Barrientos" ],
+      [ "Барфилд", "Barfield" ],
+      [ "Бас", "Bass" ],
+      [ "Бассетт", "Bassett" ],
+      [ "Баумана", "Bauman" ],
+      [ "Бегай", "Begay" ],
+      [ "Бейн", "Bain" ],
+      [ "Беланжер", "Belanger" ],
+      [ "Уайт", "White" ],
+      [ "Бердик", "Burdick" ],
+      [ "Берк", "Burk" ],
+      [ "Беркетт", "Burkett" ],
+      [ "Беркс", "Burks" ],
+      [ "Беркхарт", "Burkhart" ],
+      [ "Бернар", "Bernard" ],
+      [ "Бернс", "Burns" ],
+      [ "Бернье", "Bernier" ],
+      [ "Берр", "Burr" ],
+      [ "Беррис", "Burris" ],
+      [ "Берт", "Burt" ],
+      [ "Бертон", "Burton" ],
+      [ "Биби", "Beebe" ],
+      [ "Бизли", "Beasley" ],
+      [ "Бёрч", "Birch" ],
+      [ "Бланд", "Bland" ],
+      [ "Блаунт", "Blount" ],
+      [ "Блек", "Black" ],
+      [ "Блидсо", "Bledsoe" ],
+      [ "Блэкбёрн", "Blackburn" ],
+      [ "Бин", "Bean" ],
+      [ "Баттл", "Battle" ],
+      [ "Боннэр", "Bonner" ],
+      [ "Борхес", "Borges" ],
+      [ "Ботельо", "Botelho" ],
+      [ "Боуг", "Baugh" ],
+      [ "Бошан", "Beauchamp" ],
+      [ "Брайс", "Brice" ],
+      [ "Брантли", "Brantley" ],
+      [ "Бранхам", "Branham" ],
+      [ "Брасвелл", "Braswell" ],
+      [ "Брей", "Bray" ],
+      [ "Брейди", "Brady" ],
+      [ "Бренд", "Brand" ],
+      [ "Брубейкер", "Brubaker" ],
+      [ "Бруссар", "Broussard" ],
+      [ "Брэннон", "Brannon" ],
+      [ "Бургер", "Burger" ],
+      [ "Бурлесон", "Burleson" ],
+      [ "Куик", "Quick" ],
+      [ "Бьюли", "Beaulieu" ],
+      [ "Бэбб", "Babb" ],
+      [ "Бэрри", "Berry" ],
+      [ "Бэррон", "Barron" ],
+      [ "Бёрден", "Burden" ],
+      [ "Бёрдетт", "Burdette" ],
+      [ "Бёрнем", "Burnham" ],
+      [ "Вайз", "Wise" ],
+      [ "Вайзмен", "Wiseman" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Ванхорн", "Vanhorn" ],
+      [ "Ватсон", "Watson" ],
+      [ "Ватт", "Watt" ],
+      [ "Вейр", "Ware" ],
+      [ "Джолли", "Jolly" ],
+      [ "Хэм", "Ham" ],
+      [ "Виггинс", "Wiggins" ],
+      [ "Видаль", "Vidal" ],
+      [ "Виджид", "Vigil" ],
+      [ "Виейра", "Vieira" ],
+      [ "Вишневский", "Wisniewski" ],
+      [ "Дэнт", "Dent" ],
+      [ "Волк", "Wolf" ],
+      [ "Уилл", "Will" ],
+      [ "Вомак", "Womack" ],
+      [ "Гейтс", "Gates" ],
+      [ "Ист", "East" ],
+      [ "Хармс", "Harms" ],
+      [ "Вульф", "Wolfe" ],
+      [ "Вутен", "Wooten" ],
+      [ "Вьюер", "Weaver" ],
+      [ "Вэлш", "Welsh" ],
+      [ "Уайатт", "Wyatt" ],
+      [ "Габриэль", "Gabriel" ],
+      [ "Гавани", "Havens" ],
+      [ "Галант", "Gallant" ],
+      [ "Галлардо", "Gallardo" ],
+      [ "Галлоуэй", "Galloway" ],
+      [ "Гальвез", "Galvez" ],
+      [ "Ганье", "Gagne" ],
+      [ "Гарза", "Garza" ],
+      [ "Гарланд", "Garland" ],
+      [ "Гарретт", "Garrett" ],
+      [ "Гейгера", "Geiger" ],
```

Comment:
Like in the case of `Bauman` before, this is an error of context, rather than translation exactly. This would be correct in the case of, say, "Geiger counter" ("Счётчик Гейгера"). Otherwise, it should be `Гейгер`.